ui/app/templates/components/oidc/key-form.hbs

```diff
@@ -30,46 +36,48 @@
     {{/each}}
   </div>
   {{! RADIO CARD + SEARCH SELECT }}
-  <div class="box is-sideless is-fullwidth is-marginless has-top-padding-xxl">
-    <Hds::Text::Display @tag="h2" @size="400">Allowed applications</Hds::Text::Display>
-    <div class="is-flex-row">
-      <RadioCard
-        data-test-oidc-radio="allow-all"
-        @title="Allow every application to use"
-        @description="All applications can use this key for authentication requests."
-        @icon="globe"
-        @value="allow_all"
-        @groupValue={{this.radioCardGroupValue}}
-        @onChange={{this.handleClientSelection}}
-      />
-      <RadioCard
-        data-test-oidc-radio="limited"
-        @title="Limit access to selected application"
-        @description="Only selected applications can use this key for authentication requests."
-        @icon="globe-private"
-        @value="limited"
-        @groupValue={{this.radioCardGroupValue}}
-        @onChange={{this.handleClientSelection}}
-        @disabled={{@model.isNew}}
-        @disabledTooltipMessage="This option has been disabled for now. To limit access, you must first create an application that references this key."
-      />
+  {{#unless @limitOptions}}
```

Comment:
Again, conflicting feelings. Maybe it just needs to be renamed, since it's a pretty specific scenario?

Reply:
Good idea. It's tough to know how much refactor to do inside this component. But this modal situation will be re-used for Azure and GCP, so maybe a rename of the param. @hashishaw how do you feel about `@isModalDisplay`?

Reply:
I think the header/breadcrumbs should be moved to the route component. But I think it makes sense to use an arg to hide the additional form fields. I think this arg can be updated to your suggestion or `@isModalForm`

Also a nit, but maybe the `{{#unless}}` could go above the comment on line 38? I feel like it makes more sense to have the comment directly above the components it's explaining